src/backend/src/services/onboarding.services.ts

```diff
@@ -0,0 +1,40 @@
+import { Checklist, Team_Type } from '@prisma/client';
+import prisma from '../prisma/prisma';
+import { HttpException } from '../utils/errors.utils';
+
+export default class OnboardingServices {
+  /**
+   * Gets all checklists for the given user Id.
+   * @param userId the user id to get checklists for
+   * @returns all checklists for the given user Id
+   */
+  static async getUsersChecklists(userId: string) {
+    const generalChecklists = await prisma.checklist.findMany({
+      where: { teamTypeId: null }
+    });
```

Comment:
This looks great! Just also add a check to make sure date deleted is null as well